README.md

```diff
@@ -313,6 +313,7 @@ default values.
 | `conjur.org/conjurTokenReceivers` | Comma-separated list of the names of containers, in the pod, that will be injected with `conjur-access-token` VolumeMounts. (e.g. `app-container-1,app-container-2`)                  |  `nil` (only applies to authenticator) |
 | `conjur.org/containerMode` | Sidecar Container mode (`init` or `sidecar`)                  |  `nil` (only applies to authenticator) |
 | `conjur.org/containerName` | Sidecar Container name                  |  `nil` (only applies to authenticator)                              |
+| `conjur.org/container-image` | Sidecar Container image      | defaults to the value configured for the sidecar-injector at startup, using the `-secretless-image` or `-authenticator-image` CLI arguments. |
```

Comment:
I have the same feedback for this config annotation as I do for the `--xyz-image` CLI arguments, so maybe there's a broad justification that I'm not aware of.

It seems like it might be valuable to configure the image and tag separately. A user will probably want to use our default images, and only change the version, so maybe that could be a direct option.

Maybe something like `conjur.org/container-image` defines the image and `conjur.org/container-tag` defines the tag version.

Reply:
Also, the other annotations are in camelCase, so this new one should be `conjur.org/containerName`

Reply:
> Also, the other annotations are in camelCase, so this new one should be `conjur.org/containerName`

The camelCase is being depreciated to be consistent with secrets provider, so I updated the other annotations except for the conjurAuth and conjurConn which are being depreciated.

Reply:
> Maybe something like `conjur.org/container-image` defines the image and `conjur.org/container-tag` defines the tag version.

I think that is a good idea, but it deviates from the spec. As our acting PO @jtuttle do you agree with this change?